calendar/calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarAction.java

```diff
@@ -6846,6 +6846,9 @@ protected void buildViewCalendarContext(VelocityPortlet portlet, Context context
 		context.put("isDefaultView", isDefaultView(state, ToolManager.getCurrentPlacement()));
 		context.put("defaultSubview", ToolManager.getCurrentPlacement().getPlacementConfig().getProperty(PORTLET_CONFIG_DEFAULT_SUBVIEW));
 		context.put("isUpdater", SiteService.allowUpdateSite(ToolManager.getCurrentPlacement().getContext()));
+		context.put("aspectRatio", ServerConfigurationService.getString("calendar.fullCalendar.aspectRatio", "1.35"));
+		context.put("scrollTime",ServerConfigurationService.getString("calendar.fullCalendar.scrollTime", "06:00:00"));
```

Comment:
This is a minor suggestion, moving these settings to be initialized as part of the class since they likely won't change and creating a static for the config names.

Reply:
I think I have done it as you say, please take a look. Thanks!